lua/catppuccin/groups/integrations/copilot_vim.lua

```diff
@@ -0,0 +1,10 @@
+local M = {}
+
+function M.get()
+	return {
+		CopilotSuggestion = { fg = C.subtext0 },
```

Comment:
I've had some discussion with some other maintainers and we'll want to change this / standardise on `overlay0` for AI/Copilot inline completions.

Reply:
Good idea, I updated it.